donjo-app/models/User_model.php

```diff
@@ -332,44 +337,45 @@ private function sterilkan_input($post)
 	 */
 	public function update($idUser)
 	{
-		$this->session->error_msg = NULL;
-		$this->session->success = 1;
+		$_SESSION['error_msg'] = NULL;
+		$_SESSION['success'] = 1;
 
 		$data = $this->sterilkan_input($this->input->post());
 
 		if (empty($idUser))
 		{
-			$this->session->error_msg = ' -> Pengguna tidak ditemukan datanya.';
-			$this->session->success = -1;
+			$_SESSION['error_msg'] = ' -> Pengguna yang hendak Anda ubah tidak ditemukan datanya.';
+			$_SESSION['success'] = -1;
 			redirect('man_user');
 		}
 
+
 		if (empty($data['username']) || empty($data['password'])
 		|| empty($data['nama']) || !in_array(intval($data['id_grup']), range(1, 4)))
 		{
-			$this->session->error_msg = ' -> Nama, Username dan Kata Sandi harus diisi';
-			$this->session->success = -1;
+			$_SESSION['error_msg'] = ' -> Nama, Username dan Password harus diisi';
+			$_SESSION['success'] = -1;
```

Comment:
@fusionid , yg kita ingin gunakan justru adalah bentuk `$this->session->error_msg` . Mohon tidak diubah balik ke bentuk `$_SESSION['error_msg']` yg tidak kita inginkan lagi...

Reply:
Atau apakah file ini salah ubah? Sepertinya anda mengembalikan ke versi sebelumnya?
Mohon ambil lagi  file ini dari `master`...

Reply:
Aneh waktu merge master ke branch tidak ada pesan conflict, begitupun waktu push new commit ke branch, status tidak ada resolve conflict. Tapi okelah sudah di commit ulang.